lang/cs/Org.Apache.REEF.Client/AzureBatch/AzureBatchDotNetClient.cs

```diff
@@ -27,13 +27,12 @@
 using Org.Apache.REEF.Client.AzureBatch.Storage;
 using Org.Apache.REEF.Client.AzureBatch;
 using Org.Apache.REEF.Client.AzureBatch.Util;
-using System.IO;
 
 namespace Org.Apache.REEF.Client.DotNet.AzureBatch
 {
     class AzureBatchDotNetClient : IREEFClient
     {
-        private static readonly Logger Log = Logger.GetLogger(typeof(AzureBatchDotNetClient));
+        private static readonly Logger Logger = Logger.GetLogger(typeof(AzureBatchDotNetClient));
```

Comment:
To be consistent with the other classes use LOG. Also, try not to name a variable the same name and case the class; it causes confusion.

Reply:
will fix it.